src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java

```diff
@@ -193,69 +194,159 @@ public class LaDigitalAssister extends FlowInputs {
   
   private String personalSituationDisability;
 
+  // Income
+  private String householdSearchingForJob;
+
+  @NotEmpty(message="{error.missing-general}")
+  private List<String> jobSearch;
+
+  private String workDisqualificationInd;
+
+  private String selfEmploymentIncome;
+
+  @NotBlank(message="{error.missing-general}")
+  private String householdMemberJobAdd;
 
-  @Money
-  @NotBlank
+  @NotBlank(message="{error.missing-general}")
+  private String employerName;
+
+  private String selfEmployed;
+
+  private String jobPaidByHour;
+
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String hourlyWage;
 
-  @Range(min=1, max=100)
-  @NotBlank
+  @Range(message="{error.invalid-range}", min=1, max=100)
+  @NotBlank(message="{error.missing-general}")
   private String hoursPerWeek;
 
-  @NotBlank
+  @NotBlank(message="{error.missing-pay-period}")
   private String payPeriod;
 
-  @Money
-  @NotBlank
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String payPeriodAmount;
 
-  private String additionalIncome;
+  @NotEmpty
+  private List<String> additionalIncome;
 
-  @Money
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String moneyOnHand;
-  
+
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String monthlyHouseholdIncome;
   
   private String switchToIncomeByJob;
 
+  @NotEmpty
   private List<String> householdHomeExpenses;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesRent;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesMortgage;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesHomeownerInsurance;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesPropertyTax;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesCondoFees;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesLotRental;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesFloodInsurance;
 
+  @Money(message="{error.invalid-money}")
+  @NotBlank(message="{error.missing-dollar-amount}")
   private String expensesOther;
 
+  @NotEmpty
```

Comment:
We might want a better error message for these. Right now it just does this on any @NotEmpty field: 
<img width="613" alt="Screenshot 2023-12-08 at 9 55 36 AM" src="https://github.com/codeforamerica/la-doc-uploader/assets/7966375/5e97278e-d5c3-4326-a563-ebddde189c3f">

Might be out of scope of this PR though.

Reply:
Looks like the default comes from the jakarta library, maybe we can change the default override in the FFB itself as well